### PR TITLE
add simple card renderer

### PR DESCRIPTION
**Pull request by MichalMikolajewicz**

### CODE REVIEW

File: Assets/Scripts/Card.cs [hunk 1]
```diff
@@ -1,22 +1,56 @@
...
     {
-
+        UpdateSprite();
     }
-
     void Update()
     {
+        UpdateSprite();
+    }
+    void UpdateSprite()
```
Comment (Ko4s) [inline Assets/Scripts/Card.cs:33]: WDYT about changing name to `FlipCard` or something ?

**Review by szymongabrek — APPROVED** [review]
Nice atlas ;)

File: Assets/Scripts/Game.cs [hunk 2]
```diff
@@ -4,4 +4,15 @@
 
 public class Game : MonoBehaviour
 {
+    public GameObject CardPrefab;
```
Comment (Ko4s) [inline Assets/Scripts/Game.cs:7]: imo `[SerializeField]` is better approach

File: Assets/Scripts/Game.cs [hunk 3]
```diff
@@ -4,4 +4,15 @@
...
+
+
+    void Start()
+    {
+        Instantiate(CardPrefab, new Vector3(0, 0, 0), Quaternion.identity);
+    }
+
+    void Update()
+    {
+    }
```
Comment (Ko4s) [inline Assets/Scripts/Game.cs:17]: I'd remove Updates if there are not used yet

File: Assets/Scripts/Game.cs [hunk 4]
```diff
@@ -4,4 +4,15 @@
 
 public class Game : MonoBehaviour
 {
+    public GameObject CardPrefab;
+
+
+    void Start()
+    {
+        Instantiate(CardPrefab, new Vector3(0, 0, 0), Quaternion.identity);
+    }
```
Comment (Ko4s) [inline Assets/Scripts/Game.cs:13]: WDYT about using Awake method for Initiating functions ?

Comment (MichalMikolajewicz) [reply Assets/Scripts/Game.cs:13]: Only for testing purpose

File: Assets/Scripts/Game.cs (hunk 2 above)
Comment (MichalMikolajewicz) [reply Assets/Scripts/Game.cs:7]: Will check

File: Assets/Scripts/Card.cs (hunk 1 above)
Comment (MichalMikolajewicz) [reply Assets/Scripts/Card.cs:33]: Yea, only for testing purpose

File: Assets/Scripts/Card.cs [hunk 5]
```diff
@@ -1,22 +1,56 @@
...
 using UnityEngine;
+using UnityEngine.U2D;
 
 public class Card : MonoBehaviour
 {
     public Suits Suit = Suits.NoSuits;
     public Ranks Rank = Ranks.NoRanks;
+    public SpriteRenderer spriteRenderer;
+    public Sprite newSprite;
+    public SpriteAtlas Atlas;
```
[the remark this answers is not in the document]
Comment (MichalMikolajewicz) [reply Assets/Scripts/Card.cs:14]: Yes, at least for now

**Pull request merged**